docs/views.md

```diff
@@ -188,11 +190,11 @@ aren't unique problems, and can be solved incrementally.
   - using a configuration service / key management service for holding and
     managing private/public keys used by the system
 
-- Fighting conway's law (todo: fill up the spiel here about fighting company org
-  structures)
+- Fighting conway's law (There's a whole spiel here about fighting company org
```

Comment:
perhaps a link to: https://www.thoughtworks.com/insights/blog/applying-conways-law-improve-your-software-development

Reply:
I'll add a link to the wikipedia article, seems clearer and more references